stdlib/private/OSLog/CMakeLists.txt

```diff
@@ -8,6 +8,12 @@ endif()
 if (SWIFT_ENABLE_EXPERIMENTAL_STRING_PROCESSING)
   list(APPEND swift_oslog_darwin_dependencies "_StringProcessing")
 endif()
+if((SWIFT_BUILD_CLANG_OVERLAYS
```

Comment:
Is it worth adding a comment on these that they use Darwin and Darwin re-exports _Builtin_float? Ideally we would stop using the `Darwin` module from the Swift standard libraries, as it brings in a _ton_ of dependencies and there are much smaller finer grained modules under it. Although maybe that's a moot point until we can make macOS 15 et al the lowest supported version.

Reply:
Yes, it's better to add more context in the code itself, and not rely on the terse commit message alone.